language/de/cli.php

```diff
@@ -160,6 +162,10 @@
 
 	'CLI_USER_ADD_SUCCESS'		=> 'Benutzer %s erfolgreich angelegt.',
 	'CLI_USER_DELETE_CONFIRM'	=> 'Soll „%s“ wirklich gelöscht werden? [y/N]',
+	'CLI_USER_DELETE_ID_CONFIRM'	=> 'Bist du sicher, dass du die Benutzer mit den IDs „%s“ löschen möchtest? [y/N]',
+	'CLI_USER_DELETE_ID_SUCCESS'	=> 'Die Benutzer-IDs wurden erfolgreich gelöscht.',
+	'CLI_USER_DELETE_ID_START'		=> 'Löschen von Benutzern anhand der Benutzer-ID',
```

Comment:
Muss hier "Benutzer-IDs" nicht Plural sein? Weil mehrere Benutzer und dazu mehrere IDs gehören?

Reply:
Der Erfolgssatz wird aber geändert so dass keine IDs mehr vorkommen... In meinen Augen wäre das schon so ok. Oder aber "Löschen von Benutzern anhand ihrer Benutzer-IDs"?

Reply:
"Löschen von Benutzern anhand ihrer Benutzer-IDs" - ich denke, das war gemeint.

Reply:
Ja, so ist der Bezug klar.